plugins/cache_promote/cache_promote.cc

```diff
@@ -342,6 +498,14 @@ class LRUPolicy : public PromotionPolicy
   LRUMap _map;
   LRUList _list, _freelist;
   size_t _list_size = 0, _freelist_size = 0;
+
+  // internal stats ids
+  int freelist_size_id = -1;
```

Comment:
As above, I think we should post-fix all these with _id.

Reply:
@zwoop changed and squashed.